src/js/game/Entities/Sheep.js

```diff
@@ -2,12 +2,14 @@ const BaseEntity = require("./BaseEntity.js");
 const EventType = require("../Event/EventType.js");
 
 module.exports = class Sheep extends BaseEntity {
-    constructor(controller, type, identifier, x, y, facing) {
+    constructor(controller, type, identifier, x, y, facing, skipView = false) {
```

Comment:
Rather than passing in this new param, can we skip `prepareSprite()` when `this.controller.levelView` is undefined?

Reply:
👌